File: packages/templates/python-fastapi-standalone/src/app/routes/health.py
from datetime import datetime, timezone
from fastapi import APIRouter
from pydantic import BaseModel

router = APIRouter()


class HealthResponse(BaseModel):
    status: str
    service: str
    timestamp: str


@router.get("/health", response_model=HealthResponse)
async def health_check():
    return HealthResponse(
        status="ok",
        service="{{projectName}}",
        timestamp=datetime.now(timezone.utc).isoformat(),
    )

File: packages/templates/stacks/node-fastify/src/routes/health.ts
import type { FastifyPluginAsync } from 'fastify';

export const healthRoutes: FastifyPluginAsync = async (fastify) => {
  fastify.get('/health', async (_request, _reply) => {
    return {
      status: 'ok',
      service: '{{projectName}}',
      timestamp: new Date().toISOString(),
    };
  });
};
